_notebooks/textbook.py

```diff
@@ -130,3 +144,22 @@ def clean_raw_text(self, disable_pysbd=False):
             if not disable_pysbd:
                 clean_text = "\n".join(seg.segment(clean_text))
             chapter.clean_text = clean_text
+    
+    def resolve_coreference(self):
```

Comment:
I think the functions which work on specific chapters at a time be implemented in the `Chapter` class, right? Much better to update the chapter attributes (variables) from within the class then here?

We can still keep the driver loop, which iterates over all chapters over here.

Reply:
We aren't using it chapter wise right? Wouldn't we want to do coref-resolution for everything?

Reply:
We should prefer a chapter wise implementation, since that allows us to debug and iterate on smaller units of text which we can read and inspect manually.